pkg/diagnostics/metrics.go

```diff
@@ -34,3 +39,9 @@ func MetricsHTTPMiddleware(next fasthttp.RequestHandler) fasthttp.RequestHandler
 		http_prometheus.WithPathLabel()))
 	return fasthttpadaptor.NewFastHTTPHandler(mw(nethttpadaptor.NewNetHTTPHandlerFunc(log, next)))
 }
+
+// InitMetrics initializes metrics
+func InitMetrics(appID string) error {
+	err := DefaultMonitoring.Init(appID)
```

Comment:
nit: can this be `return DefaultMonitoring.Init(appID)`?

Reply:
I will add additional metrics initialization code... this is WIP PR :)